app/controllers/podcasts_controller.rb

```diff
@@ -9,6 +9,9 @@ def index
   # GET /podcasts/1
   def show
     authorize @podcast
+
+    @recently_published = @podcast.episodes.published.first
+    @next_scheduled = @podcast.episodes.where("released_at > ?", Time.now).order(:released_at).first
```

Comment:
Might be worth making this a scope?  `@podcast.episodes.scheduled.first`.  I imagine we'll need that scope elsewhere.